faqs/case-study-1.js

```diff
@@ -0,0 +1,283 @@
+const caseStudyOne = [
```

Comment:
Can't we rename this to something meaningful? Like `LeadGenerationCaseStudy` or `FormesterOverviewCaseStudy` or may be something more relevant.